lib/ansible/module_utils/cloudfront.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since module_utils/aws/ has been merged this could be moved to module_utils/aws/cloudfront.py now